community/fvwm/APKBUILD

```diff
@@ -2,9 +2,9 @@
 # Maintainer: ScrumpyJack <scrumpyjack@st.ilet.to>
 pkgname=fvwm
 pkgver=2.6.7
-pkgrel=0
+pkgrel=1
 pkgdesc="FVWM is a powerful desktop window manager for the X  Window system"
-url="http://fvwm.org"
```

Comment:
This URL is still valid, and seems to be the home page of the project. Why would we want to show the source code browser instead of the real homepage?